tests/narrative_proxy.py

```diff
@@ -6,7 +6,10 @@
 
 import websockets
 
-from tests.narrative import InteractionDirection, _Narrative
+from ert_shared.ensemble_evaluator.narratives.narrative import (
+    InteractionDirection,
+    _Narrative,
+)
 
 
 class NarrativeProxy(object):
```

Comment:
Should probably move this file as well. To `ert_shared.ensemble_evaluator.narratives.proxy`